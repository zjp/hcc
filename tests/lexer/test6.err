FATAL [1,1]: Integer literal too large;  using max value
FATAL [1,14]: Integer literal too large;  using max value
