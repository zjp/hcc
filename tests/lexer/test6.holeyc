123423452345 123452345234523452345234523452345345 1234

