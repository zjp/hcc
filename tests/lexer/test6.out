INTLIT:2147483647 [1,1]
INTLIT:2147483647 [1,14]
INTLIT:1234 [1,51]
EOF [3,1]
